src/App.vue

```diff
@@ -287,12 +296,27 @@ export default class App extends Mixins(
 
   /** Data for fee summary component. */
   private get feeFilingData (): Array<FilingDataIF> {
-    return this.getFilingData
-      ? [{ ...this.getFilingData,
-        futureEffective: this.getEffectiveDateTime.isFutureEffective,
-        priority: this.getStaffPaymentStep.staffPayment.isPriority,
-        waiveFees: (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE) }]
-      : []
+    let filingData: Array<FilingDataIF> = []
+    if (this.getFilingData) {
+      filingData = cloneDeep(this.getFilingData)
+      if (this.getEntityType === CorpTypeCd.COOP && this.getFilingData.length) {
+        // Only set Future Effective and Priority to Special Resolution Fee
+        const filteredFilingData = filingData.filter(x => x.filingTypeCode === FilingCodes.SPECIAL_RESOLUTION)
```

Comment:
Try using `filingData.find()` to return an element.

Reply:
makes sense. Will do it!